tests/unit/plugins/modules/test_kubevirt_vm.py

```diff
@@ -65,6 +65,46 @@ def vm_definition_create():
     }
 
 
+@pytest.fixture(scope="module")
+def vm_definition_running():
+    return {
+        "apiVersion": "kubevirt.io/v1",
+        "kind": "VirtualMachine",
+        "metadata": {
+            "name": "testvm",
+            "namespace": "default",
+        },
+        "spec": {
+            "running": True,
+            "template": {
+                "spec": {
+                    "domain": {"devices": {}},
+                },
+            },
+        },
+    }
+
+
+@pytest.fixture(scope="module")
+def vm_definition_stopped():
+    return {
+        "apiVersion": "kubevirt.io/v1",
+        "kind": "VirtualMachine",
+        "metadata": {
+            "name": "testvm",
+            "namespace": "default",
+        },
+        "spec": {
+            "running": False,
+            "template": {
+                "spec": {
+                    "domain": {"devices": {}},
+                },
+            },
+        },
+    }
+
+
```

Comment:
We can reduce this code extracting the common parts such:

```python
@pytest.fixture(scope="module")
def vm_definition_base():
    return {
        "apiVersion": "kubevirt.io/v1",
        "kind": "VirtualMachine",
        "metadata": {
            "name": "testvm",
            "namespace": "default",
        },
        "spec": {
            "template": {
                "spec": {
                    "domain": {"devices": {}},
                },
            },
        },
    }


@pytest.fixture(scope="module")
def vm_definition_running(vm_definition_base):
     vm = vm_definition_base
     vm["spec"]["running"] = True
     return vm



@pytest.fixture(scope="module")
def vm_definition_stopped(vm_definition_base):
    vm = vm_definition_base
    vm["spec"]["running"] = True
    return vm
```
WDYT?

Reply:
Tried something similar. Unfortunately it leads to issues where the order of values created in the YAML by `yaml.dump` changes.

Reply:
I see. I was able to see the same behavior.